.github/workflows/ci.yml

```
@@ -39,6 +39,14 @@ jobs:
 
       - run: bun install
 
+      - name: Generate bindings
+        run: |
+          bun run generate
+          if [[ $(git diff --stat src/binding.ts) != '' ]]; then
+              echo "Bindings changed but was not committed. Re-generate and commit the new binding.ts!"
+              exit 1
```

to see what's the diff

```suggestion
          if [[ $(git diff --stat src/binding.ts) != '' ]]; then
              git --no-pager diff
              echo "Bindings changed but was not committed. Re-generate and commit the new binding.ts!"
              exit 1
```